src/main/resources/db/migration/V1_29__add_source_id_to_dwp_enrichment_fields.sql

```diff
@@ -0,0 +1,6 @@
+INSERT INTO acquirer_subscription_enrichment_field (acquirer_subscription_id, enrichment_field)
+SELECT asub.id, 'SOURCE_ID'
+FROM acquirer_subscription asub join acquirer a on asub.acquirer_id = a.id
+WHERE asub.event_type = 'DEATH_NOTIFICATION' and a.name like '%DWP%' and not exists (
+        select 1 from acquirer_subscription_enrichment_field asube where asube.acquirer_subscription_id = asub.id and asube.enrichment_field = 'SOURCE_ID'
+    );
```

Comment:
So I am not sure if it is best for this or to just manually add it in, happy for whichever but worth thinking that we are gonna flatten these migrations in a bit :)

Reply:
I' leave it in for now